docs/dev/introduction_to_module_serialization.rst

```diff
@@ -0,0 +1,206 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Introduction to Module Serialization
+====================================
+**Author**: `Zhao Wu <https://github.com/FrozenGene>`_
```

Comment:
Given that this is part of doc as opposed to tutorial, let us not include author's name for now(to be consistent with the rest part of the doc) as we want to encourage future collaborations and improvements. Of course the contribution is part of the git commit history :)

Reply:
Done.